src/handlers/sra_handlers.ts

```diff
@@ -77,19 +77,26 @@ export class SRAHandlers {
         res.status(HttpStatus.OK).send(orderbookResponse);
     }
     public async postOrderAsync(req: express.Request, res: express.Response): Promise<void> {
+        const shouldSkipValidation = req.query.skipValidation === 'true';
         schemaUtils.validateSchema(req.body, schemas.sraPostOrderPayloadSchema);
         const signedOrder = unmarshallOrder(req.body);
         if (WHITELISTED_TOKENS !== '*') {
             const allowedTokens: string[] = WHITELISTED_TOKENS;
             validateAssetTokenOrThrow(allowedTokens, signedOrder.makerToken, 'makerToken');
             validateAssetTokenOrThrow(allowedTokens, signedOrder.takerToken, 'takerToken');
         }
+        if (shouldSkipValidation) {
+            res.status(HttpStatus.OK).send();
```

Comment:
Would this close the connection for the client? don't we need something like `close()`? or does `status()` close the connection too?

Reply:
Nah, `send()` calls `end()` as well. We use `send()` instead of `end()` as a pattern because it generates an ETag header for cache validation.

Reply:
Thanks for clarifying